README.md

```diff
@@ -1,7 +1,25 @@
-# Docker-DB Test Environment Starter
+# Integration Test Docker Environment
 
-**This repository is under development and might undergo breaking changes. 
-Use it only, if you know what you are doing.**
+## About
+
+This project provides a command line interface to start a test environment with an [Exasol Docker DB](https://hub.docker.com/r/exasol/docker-db). It starts an Exasol Docker-DB container and an associated test container where Exaplus and ODBC is already installed. Both containers exist in the same Docker network. This allows you to connect from the test container to the Docker-DB container. Further more, the database gets populated with some test data. You can find the test data under `tests/test/enginedb_small/import.sql`. Besides the test container, you can also access the Exasol database and the Bucket-FS from the host via port forwards. 
+
+### Comparison to Docker-DB and the Exasol Testcontainers
+
+* This project uses the [Docker-DB](https://hub.docker.com/r/exasol/docker-db), but it does the configuration, setup and startup for you. For example, it waits until the Database and the Bucket-FS accept connections. It also populates it with test data and provides with the test container an environment to access the database.
+* The [Exasol Testcontainers](https://github.com/exasol/exasol-testcontainers/) provides similar features for Java JUnit integration tests, so if you have an Java project use the Exasol Testcontainer, because they are more integrated in the Java ecosystem then this project. However, if you have a project in any other language you can use this project to start a test Exasol database via a commandline interface.
+
+## Prerequisites
+
+In order to start a Docker-DB Test Environment, you need:
+
+* Linux (MaxOS X and Windows is currently not supported, because the Exasol Docker-DB requires privileged mode)
```

Comment:
```suggestion
* Linux (MaxOS X and Windows are currently not supported, because the Exasol Docker-DB requires [privileged mode](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities))
```